libs/designsystem/modal/src/modal/services/can-dismiss.helper.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { createServiceFactory, SpectatorService } from '@ngneat/spectator';
+import { AlertConfig } from '../alert';
+import { CanDismissHelper } from './can-dismiss.helper';
+import { AlertHelper } from './alert.helper';
+
+describe('CanDismissHelper', () => {
+  let spectator: SpectatorService<CanDismissHelper>;
+
+  const createService = createServiceFactory({
+    service: CanDismissHelper,
+    mocks: [AlertHelper],
+  });
+
+  beforeEach(() => {
+    spectator = createService();
+  });
+
+  describe('getCanDismissCallback', () => {
+    it('should return an async function that returns true, if the provided callback return true', async () => {
+      const callback = () => true;
+
+      const canDismissCallback = spectator.service.getCanDismissCallback(callback);
+      const result = await canDismissCallback();
+
+      expect(result).toBe(true);
+    });
+
+    it('should call the showAlert method, if the provided callback return an AlertConfig', async () => {
+      //@ts-ignore
```

Comment:
What's this? 🤔

Reply:
This was previously needed because 'showAlert' was a private method and Typescript complained that is was spying on the method: `const showAlertSpy = spyOn(spectator.service, 'showAlert')`. But it `ts-ignore` comment is not needed anymore, since I updated the method to be public :) Consider it removed!